src/datasets/obsolete/retention/reference.md

```diff
@@ -1,5 +1,9 @@
 # 1 Day Retention
 
+***As of 2019-08-13, this dataset has been deprecated and is no longer
```

Comment:
How might someone calculate what was previously in the retention dataset?

Reply:
The process is involved -- it requires computing a HLL sketch per client dimension for each subsession date. The cut-off period is 10 days, which is enough time for 99% of pings the subsession date to arrive. The new-profile and main pings are merged together to form the source.

In BigQuery, this would involve cleaning the datasets and aligning the clients by a particular date, then using the `HLL_COUNT` udfs. Overall, there were only a few users, and I wouldn't recommend emulating this route due to the complexity.

Reply:
I think it is a good idea to have a simple retention query, ala https://github.com/mozilla/stmo_core_product_metrics/blob/master/fennec/android/retention_day.sql

(If that seems sufficient, feel free to point people there).